MediaPushPlugin/src/main/resources/media-push-extension/js/canvas-designer-widget.js

```diff
@@ -59,16 +59,19 @@ export function CanvasDesigner() {
         });
     }
 
-    var syncDataListener = function(data) {};
-    var dataURLListener = function(dataURL) {};
-    var captureStreamCallback = function() {};
+    var syncDataListener = function (data) {
```

Comment:
Please don't embed this file from JS SDK

Reply:
It will solved in https://github.com/ant-media/Ant-Media-Server/issues/5911